packages/mdadm/mdadm.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package %{nil}
+Name: %{_cross_os}mdadm
+Version: 4.3
+Release: 1%{?dist}
+Summary: mdadm is used for controlling Linux md devices (aka RAID arrays)
+License: GPL-2.0-only
+URL: https://cdn.kernel.org/pub/linux/utils/raid/mdadm/
+Source0: https://cdn.kernel.org/pub/linux/utils/raid/mdadm/mdadm-%{version}.tar.xz
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}systemd-devel
+
+%description
+%{summary}.
+
+%global set_env \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc \\\
+CFLAGS="%{_cross_cflags} -DNO_COROSYNC -DNO_DLM" \
```

Comment:
It looks like this Makefile is set up to use `CXFLAGS` to extend the flags.

Otherwise you'd need to pass CFLAGS on the `make` command line to override the defaults. (This is most likely why debuginfo isn't being generated.)